### DISCUSSION TITLE
make install fails

### DISCUSSION THREAD

**Discussion by apopaa** [sse-secure-systems/connaisseur#176] [Q&A] [answered]
Hi,

Is jq required? I have manually installed yq, by downloading and copying the binary, but when I run your "make install" I get a "/bin/sh: jq: not found" and then later on make fails with "fail "Please provide a name for your cluster", even though it claims that "Using 'make' sets the cluster name automatically". Does it fail to set the cluster name because it is missing jq?

**Comment by apopaa** [comment, marked as answer]
Yes, your connaisseur requires jq, in addition to yq.
Once I installed it with:
`curl -fsSL -o /usr/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && chmod +x /usr/bin/jq`
the install went smoothly.

**Reply by xopham** [reply]
Hi @apopaa , that is correct. Version 1.5.0 makes use of `jq` in the Makefile. In fact, we are removing the dependency on both `jq` and `yq` in v2.0.0 that will be released within the next weeks (hopefully earlier ;-) ). What you can do right now however is to not use the Makefile and just call:
```bash
bash helm/certs/gen_certs.sh
helm install connaisseur helm --atomic --set alerting.cluster='mycluster'  --create-namespace --namespace connaisseur
```
let me know if that works :slightly_smiling_face:
